### ISSUE TITLE
Duplicated Concentration Status added

### ISSUE THREAD

**Issue by OLStefan** [NekroDarkmoon/a5e-concentrator#1]
Activating this module adds a second Concentration status to the list. Clicking on either one of them activates both.

This happens when this module is the only one active, all other statuses (including the Concentration status with the meditating incon) are from the game system itself.

Module version: 2.0.0
A5E System version: 0.8.9

![image](https://user-images.githubusercontent.com/3206519/196139524-a3f6d699-1ce4-4e97-9fd6-e5c2534af918.png)

**Comment by NekroDarkmoon** [comment]
Yeah, this is something that needs to be fixed after the changes made to the condition stuff in the system.
